docs/_frontend/src/models/FilterForm/FilterForm.js

```diff
@@ -3,19 +3,31 @@ import './FilterForm.css';
 
 const { createElement: e } = React;
 
+const compareEditions = (a, b) => {
+  const getPriority = (edition) => {
+    const products = ['Spark NLP', 'Spark NLP for Healthcare', 'Spark OCR'];
+    const index = products.indexOf(edition);
+    if (index !== -1) {
+      return index - products.length;
+    }
+    if (edition.includes('Spark NLP for Healthcare')) {
+      return 0;
```

Comment:
The order should be the same as for products:

1. Spark NLP versions
2. Spark NLP for Healthcare versions
3. Spark OCR versions

Reply:
Ohh .. Done